bucket/filezilla.json

```diff
@@ -27,7 +32,6 @@
             "FileZilla"
         ]
     ],
-    "persist": "fzdefaults.xml",
```

Comment:
You do not persist anything. Which is nonsense.

Reply:
ok, i added the config directory